src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,155 @@
+package duke.ui;
+
+import java.util.ArrayList;
+
+import duke.task.Task;
+import duke.tasklist.TaskList;
+
+/**
+ * Ui class to perform the outputting to user
+ */
+public class Ui {
+    private static final String INTRO = "Hello! I'm Duke\nWhat can I do for you?\n";
+    private static final String OUTRO = "Bye. Hope to see you again soon!\n";
+
+    public Ui() {}
```

Comment:
As empty constructors do not need to be declared, perhaps it would be a good idea to remove this constructor?